registry/extension/oras/artifactservice.go

```diff
@@ -50,17 +55,27 @@ func (h *referrersHandler) Referrers(ctx context.Context, revision digest.Digest
 			return nil
 		}
 
-		desc, err := blobStatter.Stat(ctx, referrerRevision)
-		if err != nil {
-			return err
+		// filtering by artifact type
+		if ArtifactMan.ArtifactType() == referrerType {
```

Comment:
`referrerType` should be renamed `artifactType`. 
What is the behavior when it is empty?

Reply:
Currently there's no support if artifactType is empty. I'm assuming that if they don't specify the artifactType, it should return all the referrers of any type? I'll update it to support this.